### PR TITLE
[RW-4088][risk=no] UI updates for CB wizard

### PR DESCRIPTION
**Pull request by dolbeew**
- Increase the width of the wizard, add borders to the columns in list search results table ([RW-4088](https://precisionmedicineinitiative.atlassian.net/browse/RW-4088))
- Remove conditions on showing code in CB trees ([RW-2966](https://precisionmedicineinitiative.atlassian.net/browse/RW-2966))
- Always show Add button on attributes page, greyed out if form is invalid ([RW-3148](https://precisionmedicineinitiative.atlassian.net/browse/RW-3148))
---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx [hunk 1]
```diff
@@ -315,8 +315,7 @@ export const AttributesPage = withCurrentWorkspace() (
         }
         return acc;
       }, formErrors);
-      formValid = formValid ||
-        (this.isMeasurement && !operatorSelected && form.cat.some(attr => attr.checked));
+      formValid = formValid && !(this.isMeasurement && !operatorSelected && !form.cat.some(attr => attr.checked));
```
Comment (freemabd) [inline ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: can we add a comment explaining what case this code is validating? `!(this.isMeasurement && !operatorSelected && !form.cat.some(attr => attr.checked))`

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx [hunk 2]
```diff
@@ -499,7 +498,8 @@ export const AttributesPage = withCurrentWorkspace() (
       const {node} = this.props;
       const {calculating, count, countError, form, loading, options} = this.state;
       const {formErrors, formValid} = this.validateForm();
-      const disabled = calculating || form.exists || !formValid || form.num.every(attr => attr.operator === 'ANY');
+      const disableAdd = calculating || !formValid || (this.isMeasurement && !formValid && !form.exists);
```
Comment (freemabd) [inline ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: Why is this snippet -> `(this.isMeasurement && !formValid && !form.exists)` not included in the function `validateForm()`? Seems like this is part of form validation

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx [hunk 3]
```diff
@@ -499,7 +498,8 @@ export const AttributesPage = withCurrentWorkspace() (
       const {node} = this.props;
       const {calculating, count, countError, form, loading, options} = this.state;
       const {formErrors, formValid} = this.validateForm();
```
Comment (freemabd) [inline ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: Why do we have formErrors and formValid? Seems like these could get out of sync in the validateForm function? Can you get away with 1 const like fromErrors? If formErrors is empty the the form is valid?

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx (hunk 1 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: Good idea, done. Having these domain-specific conditions can get confusing.

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx (hunk 3 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: We don't show an error for an empty input unless it's already been typed in, so it is possible for the form to be invalid and have an empty error array.

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx (hunk 2 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: Good catch. This is actually already checked in `validateForm()` now that I fixed the condition there, so I just removed this snippet and it works.

**Pull request merged**